finality-aleph/src/crypto.rs

```diff
@@ -172,6 +172,19 @@ impl MultiKeychain for KeyBox {
     }
 }
 
+/// Old format of signatures, needed for backwards compatibility.
+#[derive(PartialEq, Eq, Clone, Debug, Decode, Encode)]
+pub(crate) struct SignatureV1 {
```

Comment:
isn't it too general? I mean, next time we have similar situation, we would have to put some structs to such module, somehow suggesting that they were connected with `Signature`

Reply:
well if they belonged to `crypto.rs` they should also belong to `crypto::vX` also. Or am I missing something?

Reply:
oh, I missed that you wanted to do a module within `crypto` module

okay, that makes more sense; anyway, I do not have a strong opinion; up to @DamianStraszak

Reply:
Just to make sure I understand: you would keep the old names `Signature`, `AlephJustification` but put them in crypto::v1 and justification::v1 instead?

Reply:
Either way is fine for me, during import we can always rename them to `SignatureV1`

Reply:
Ok, I went for `v1::SignatureV1` because otherwise the code within the `v1` mod is a bit confusing.

Reply:
I'm not convinced this helps with anything, especially with the name containing `V1` in it anyway. I would suggest going back to the version without the submodule, unless there is some reason I'm missing.

Reply:
Ok, rolled back. I think this would make more sense if the `V1` suffix was not there, but then I hate to have two things with the same name in a single file. Anyway, no modules `v1` :P